examples/light-switch-app/light-switch-common/light-switch-app.zap

```diff
@@ -4400,29 +4400,29 @@
     },
     {
       "id": 2,
-      "name": "MA-onofflightswitch",
+      "name": "MA-dimmablelight",
```

Comment:
Are we use this is correct? it should be a dimmable light swithc iirc

Reply:
Yes, it is suggested in the design document that we'll make the endpoint1 type as dimmable light

Reply:
It is a dimmer-switch. I've named it as dimmable light as per the discussion in design page..
please let me know if it needs to be updated as dimmable light switch

Reply:
Outside of the design page, is it logical for this device to be a `dimmabe light`?
Also, where in the design page does it say that the this device should be a `dimmable light`?